e2e/tests/moveObjects.e2e.spec.js

```diff
@@ -27,12 +27,46 @@ This test suite is dedicated to tests which verify the basic operations surround
 const { test, expect } = require('@playwright/test');
 
 test.describe('Move item tests', () => {
-    test.fixme('Create a basic object and verify that it can be moved to another Folder', async ({ page }) => {
-        //Create and save Folder
-        //Create and save Domain Object
-        //Verify that the newly created domain object can be moved to Folder from Step 1.
-        //Verify that newly moved object appears in the correct point in Tree
-        //Verify that newly moved object appears correctly in Inspector panel
+    test('Create a basic object and verify that it can be moved to another Folder', async ({ page }) => {
+
+        await page.goto('http://localhost:8080/');
+        await page.goto('http://localhost:8080/#/browse/mine?view=grid&tc.mode=fixed&tc.startBound=1651088948584&tc.endBound=1651090748584&tc.timeSystem=utc');
+
+        let folder1 = "Folder 1";
+        let folder2 = "Folder 2";
+        // Create a new folder in the root my items folder
+        await page.locator('button:has-text("Create")').click();
+        await page.locator('li.icon-folder').click();
+
+        await page.locator('text=Properties Title Notes >> input[type="text"]').click();
+        await page.locator('text=Properties Title Notes >> input[type="text"]').fill(folder1);
+        await Promise.all([
+            page.waitForNavigation(/*{ url: 'http://localhost:8080/#/browse/mine/fae88dc4-99bb-470d-aa2d-4aea0fd37866?tc.mode=fixed&tc.startBound=1651088948584&tc.endBound=1651090748584&tc.timeSystem=utc&view=grid' }*/),

```

Comment:
do you mind removing the autofilled /*{url: 'http:// blah blah ?
It should just be the plain `page.waitForNavigation()` method

Reply:
Done!